src/fzx/tui/term_app.cpp

```diff
@@ -129,6 +155,8 @@ void TermApp::redraw()
       if (highlighted) {
         mTTY.put("\x1B[0m"sv);
       }
+    } else {
+      mTTY.put("  ", maxHeight - i);
```

Comment:
This can be removed, it was unconditionally printing those two spaces before because I got lazy, but it doesn't actually do anything
```suggestion
```